pages/1.13/security/ent/ldap/ldap-auth/index.md

```diff
@@ -9,70 +9,151 @@ enterprise: true
 <!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
 
 
-In this section you will set up the authentication method and parameters for your LDAP directory.
+In this section you will set up the authentication method and parameters for your LDAP directory. There are two bind types and two authentication methods you can use.
 
-# Selecting the authentication method
+# Bind Types
 
-1. Once you have finished specifying your connection parameters in the **Connection** tab, click **Authentication**.
+Bind operations are used to authenticate clients to the directory server,  establish an authorization identity that will be used for subsequent operations processed on that connection, and to specify the LDAP protocol version that the client will use.
 
-1.  Provide the full DN of the user account you will use to connect to the LDAP server to import users, groups, and check user credentials in the **Lookup DN** field. A few examples follow.
+There are two bind types available to you: 
 
-        cn=read-only-user,dc=los-pollos,dc=io
-        uid=read-only-user,cn=users,cn=accounts,dc=demo1,dc=freeipa,dc=org
-        uid=read-only-user,ou=users,dc=example,dc=com
+- Anonymous Bind - Uses a simple bind DN and a user DN template. Anyone can connect to the LDAP server. This bind type does not require a **Lookup DN** or **Lookup Password**. 
```

Comment:
The bind type (anonymous) is not related to the authentication method (`simple bind`). It could also use search bind.

Reply:
Thanks! I have revised this.